src/makeCalculator.js

```diff
@@ -38,6 +38,42 @@
  */
 function makeCalculator() {
   // write code here
+  const calculate = {
+    result: 0,
+
+    add(num) {
+      calculate.result += num;
+
+      return calculate.result;
+    },
+    subtract(num) {
+      calculate.result -= num;
+
+      return calculate.result;
+    },
+    divide(num) {
+      calculate.result /= num;
+
+      return calculate.result;
+    },
+    multiply(num) {
```

Comment:
добавляй відступи між методами
```suggestion
    },
    
    subtract(num) {
      calculate.result -= num;

      return calculate.result;
    },
    
    divide(num) {
      calculate.result /= num;

      return calculate.result;
    },
    
    multiply(num) {
```

Reply:
Зроблено